lychee-lib/src/filter/mod.rs

```diff
@@ -20,7 +23,7 @@ const FALSE_POSITIVE_PAT: &[&str] = &[
 /// default. This behavior can be explicitly overwritten by defining an
 /// `Include` pattern, which will match on a false positive
 pub fn is_false_positive(input: &str) -> bool {
-    input == FALSE_POSITIVE_PAT[0]
+    FALSE_POSITIVE_PAT.iter().any(|pat| input.starts_with(pat))
```

Comment:
Ah, I guess that the input was compared only to the first array entry is the reason why the SVG namespace URL was still checked: https://github.com/lycheeverse/lychee/pull/359#issuecomment-953389043

Reply:
Haha yeah. I think you're right. 😆